cfgov/v1/wagtail_hooks.py

```diff
@@ -16,15 +17,11 @@ def share_the_page(request, page):
     is_sharing = bool(request.POST.get('action-share')) and parent_page_perms.can_publish()
 
     if is_sharing or is_publishing:
-        if isinstance(page, CFGOVPage):
-            page.shared = True
+        page.shared = True
     else:
         page.shared = False
 
     page.save()
-    revision = page.save_revision()
```

Comment:
Because it was a `Page` object when it applied `page.shared = true` it wasn't actually saving this to the cfgov table since this doesn't exist in the base `Page` object. 
Same behavior with Landing, Demo pages etc.